docs/contribution.rst

```diff
@@ -0,0 +1,57 @@
+Contributions
+=============
+
+Contribution guidelines and process instructions for the ``catkit2`` package. 
+
+Guidelines
+---------
+* Keep your scope as small as possible. PRs with a smaller scope are faster to
+  contribute, and easier to review and test. Try to subdivide a goal into
+  several bite-size pieces. 
+* Use issues when starting to go out of scope: they help you keep track of
+  problems that can be adressed with future PRs. The minimum viable PR is code
+  that works, does not crash, and does not crash existing code. 
+* Avoid sneaking in changes or scope creep; if you need to change something
+  that feels unrelated it is better to open a new PR, even if it is a very
+  (very) short one. 
+* If your review process is dragging and feels slow, it is likely your PR is too
+  large in scope. 
+* Be specific when cross-linking issues/PRs in their descriptions. For example,
+  if your changes are inspired by previous code reviews, link the comment that
+  this resulted from for better context/easier paper trails. 
+
+Process
+-------
+
+
```

Comment:
Idk if these blank lines show up in the documentation; if yes, maybe reduce to one or none?

Reply:
I believe this spacing is set by the header type, but I zapped the spaces anyway. Here is a screengrab of what the spacing looks like. 
<img width="259" alt="Screen Shot 2024-05-17 at 1 36 59 PM" src="https://github.com/spacetelescope/catkit2/assets/21206463/cff9e88b-a67c-42da-b1cb-7c05e44e72d5">